packages/copy-widget-code/src/dev/textResources.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
package specific text resources should be added here (e.g. `client.actions.copy-widget-code.toasterTitle`)